notebooks/so-vits-svc-fork-4.0.ipynb

```diff
@@ -184,7 +184,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!svc train-cluster --output-path drive/MyDrive/so-vits-svc-fork/logs/44k"
+    "!svc train-cluster --output-path drive/MyDrive/so-vits-svc-fork/logs/44k/kmeans.pt"
```

Comment:
I think this place is probably unnecessary, what do you think?

Reply:
Let user decide on themselves about their kmeans file name and not to set a default filename may prevent some users who do not understand the meaning of exceptions from using it. In my opinion, It can make the project more acceptable to people and reduce feedback on ineffective issues. 😊
All in all, it depends on you, you could delete delete it in the future commit.